docs/src/pages/solid.mdx

```diff
@@ -121,6 +121,35 @@ export const uploadRouter = {
 export type OurFileRouter = typeof uploadRouter;
 ```
 
+#### Not using TypeScript V4.9 or above?
+
+Not using `v4.9` or above, meaning you won't get access to the `satisfies`
+keyword, therefore you should be using `f.router` to define a router.
+
+```ts copy filename="src/server/uploadthing.ts"
+import { createUploadthing } from "uploadthing/next";
```

Comment:
/solid?

Reply:
The import is right, it is coming from uploadthing/next